### ISSUE TITLE
TypeError exception in BaseResponse.php

### ISSUE THREAD

**Issue by ChrisHardie** [DeDmytro/laravel-cloudflare-images#3]
- Laravel Cloudflare Images version: 0.1.6
- PHP Version: 8.0.10
- Laravel/Lumen Framework Version: 8.75.0
- Database Driver & Version: MySQL 8.0.26

### Description:

Hi! Thanks for creating this package. When just getting started, I'm hitting an exception when trying to do basic things like list existing files:

```
[2021-12-23 19:08:53] local.ERROR: DeDmytro\CloudflareImages\Http\Responses\ListResponse::__construct(): Argument #1 ($result) must be of type array, null given, called in /Users/chris/develop/myproject/vendor/dedmytro/laravel-cloudflare-images/src/Http/Responses/BaseResponse.php on line 57 {"exception":"[object] (TypeError(code: 0): DeDmytro\\CloudflareImages\\Http\\Responses\\ListResponse::__construct(): Argument #1 ($result) must be of type array, null given, called in /Users/chris/develop/myproject/vendor/dedmytro/laravel-cloudflare-images/src/Http/Responses/BaseResponse.php on line 57 at /Users/chris/develop/myproject/vendor/dedmytro/laravel-cloudflare-images/src/Http/Responses/ListResponse.php:27)
```

I'm wondering if you have any guidance. Thank you.

### Steps To Reproduce:

`CloudflareApi::images()->list()`

I confirmed that these vars are set in `.env`:

```
CLOUDFLARE_IMAGES_ACCOUNT
CLOUDFLARE_IMAGES_KEY
CLOUDFLARE_IMAGES_DELIVERY_URL
CLOUDFLARE_IMAGES_DEFAULT_VARIATION
```

**Comment by ChrisHardie** [comment]
It looks like the issue is that the API key provided by Cloudflare within the Images interface is not actually able to access that API endpoint, but is instead for signing private image URLs. Instead, I had to generate an API key at https://dash.cloudflare.com/profile/api-tokens and give it edit access to the Images feature, and now I'm getting results. (I also had to make sure to run `artisan config:cache` to recreate the config cache after updating the value of `CLOUDFLARE_IMAGES_KEY`.
